app/models/work_package/pdf_export/cover.rb

```diff
@@ -60,41 +60,69 @@ def write_cover_hr
     )
   end
 
+  def cover_text_color
+    @cover_text_color ||= validate_cover_text_color
+  end
+
+  def validate_cover_text_color
+    return nil if CustomStyle.current.blank?
+
+    color = CustomStyle.current.export_cover_text_color
+    return nil if color.blank?
+
+    # pdf hex colors are defined without leading hash
+    color = color.sub('#', '')
```

Comment:
Minor remark: We have `Color#normalize_hexcode` already that does this. You could create a color instance from this input and validate it that way. This prevents some duplication here